docs/examples/steps.yaml

```diff
@@ -0,0 +1,29 @@
+---
+Steps:
```

Comment:
Could you add a step to configure Python3 please, as per Phanindra's recent ticket (DW-3763).

Reply:
Fixed by https://github.com/dwp/emr-launcher/pull/2/commits/b38c493508d143cbad8c5385f277a08c9485fc08